webapi/tct-xmlhttprequest-w3c-tests/xmlhttprequest/w3c/status-error.htm

```diff
@@ -1,60 +0,0 @@
-<!doctype html>
```

Comment:
This file is referenced by

- webapi/tct-sse-w3c-tests/tests.xml
- webapi/tct-sse-w3c-tests/tests.full.xml
- webapi/tct-sse-w3c-tests/sse-py/w3c/README
- webapi/tct-sse-w3c-tests/sse/w3c/READM

However, there is no reference in tct-xmlhttprequest-w3c-tests; please check why.